src/code/z_jpeg.c

```diff
@@ -13,32 +14,33 @@
 #define MARKER_COM 0xFE
 #define MARKER_EOI 0xD9
 
+/**
+ * Sends the JPEG decoding task to the JPEG microcode and waits for it to finish.
```

Comment:
This function creates an `OSTask`, which is responsible for sending both the microcode and the decoding task data to the RSP, and passes it on to the scheduler, so perhaps a comment like
`Schedules a JPEG decoding task and waits for it to finish.`
would be more accurate. Perhaps the function name could also better reflect that it's only scheduled at this time also?

Reply:
How about the new name and description?